src/Makevars

```diff
@@ -1,5 +1,7 @@
-
-## Use the R_HOME indirection to support installations of multiple R version
 CXX_STD = CXX11
-PKG_LIBS += $(shell ${R_HOME}/bin/Rscript -e "RcppParallel::RcppParallelLibs()")
+PKG_CXXFLAGS += -DRCPP_PARALLEL_USE_TBB=1
+
+PKG_LIBS = $(shell "${R_HOME}/bin/Rscript" -e  -e "RcppParallel:::LdFlags()") $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)
 
+PKG_LIBS += $(shell "${R_HOME}/bin/Rscript" -e  \
+              -e "RcppParallel::RcppParallelLibs()")
```

Comment:
Or maybe these new makevars fixed your bug?

Reply:
I believe that the change to `Makevars` is what allowed me to install the package on linux and other changes were made to address warnings from `devtools:check`. That said, i did this before my daughter was born and now that i'm sleep deprive my memory is not very reliable...

Reply:
Congratulations on your new family member Ignacio! Man, this was such a crazy year for you! I remember that was one of the many variables up in the air, but I didn't put together that your daughter was coming so soon!